<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseModulePrototypeTileSet</class>
 <widget class="QWidget" name="DatabaseModulePrototypeTileSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TileSetDatabaseWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="database::ui::module::PrototypeBase" name="m_pModuleBase" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>TileSet:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>61</width>
          <height>31</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="m_pHorizontalRemove">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
        <widget class="QPushButton" name="m_pHorizontalAdd">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>0</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_3">
        <property name="minimumSize">
         <size>
          <width>21</width>
          <height>61</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QPushButton" name="m_pVerticalRemove">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
        <widget class="QPushButton" name="m_pVerticalAdd">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTableWidget" name="m_pTable">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DropOnly</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>0</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>32</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>32</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>32</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>32</number>
        </attribute>
       </widget>
      </item>
     </layout>
     <zorder>frame_3</zorder>
     <zorder>m_pHorizontalRemove</zorder>
     <zorder>m_pHorizontalAdd</zorder>
     <zorder>m_pVerticalAdd</zorder>
     <zorder>frame</zorder>
     <zorder>m_pVerticalRemove</zorder>
     <zorder>m_pTable</zorder>
     <zorder>frame_3</zorder>
     <zorder>frame</zorder>
    </widget>
   </item>
  </layout>
  <zorder>m_pModuleBase</zorder>
  <zorder>groupBox</zorder>
  <zorder>m_pTable</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>database::ui::module::PrototypeBase</class>
   <extends>QWidget</extends>
   <header>databasemoduleprototypebase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
